code/datums/station_traits/neutral_traits.dm

```diff
@@ -127,3 +127,95 @@
 
 	var/new_colored_assistant_type = pick(subtypesof(/datum/colored_assistant) - get_configured_colored_assistant_type())
 	GLOB.colored_assistant = new new_colored_assistant_type
+
+/datum/station_trait/new_uniform_standards
+	name = "New Uniform Standards: Base Trait (Shouldn't Run)"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 0
+	show_in_report = TRUE
+	report_message = "We've issued some new uniforms for the crew."
+	trait_to_give = STATION_TRAIT_NEW_UNIFORM_STANDARDS
+	var/list/uniforms = list("command" = /datum/outfit/job/command_trek,
+							"engsec" = /datum/outfit/job/engsec_trek,
+							"medsci" = /datum/outfit/job/medsci_trek,
+							"srvcar" = /datum/outfit/job/srvcar_trek)
+
+/datum/station_trait/new_uniform_standards/tos
+	name = "New Uniform Standards: TOS"
+	weight = 10
```

Comment:
I have a perplexity: there are three treekie suits traits, and each one of them has a weight of 10, so the complexive weight would be 30, right? Because that would make these a bit too recurrent imo, or at least more than what the weight of a single one of them would lead you to believe: therefore I recommend you lower the weights to 3 (our pickweight procs don't support decimal, so it can't be 3.3) so that the weight of these jumpsuits as a whole is in line with similar station traits.